projects/graph/src/graph_demo.py

```diff
@@ -14,13 +14,17 @@ def main(num_vertices=8, num_edges=8):
     for num in range(num_vertices):
         # print(graph.add_vertex(str(num)))
         graph.add_vertex(str(num))
-        print(graph.vertices)
-        for _ in range(num_edges):
-            print(graph.vertices)
-            vertices = sample(graph.vertices.keys(), 2)
```

Comment:
We'll have to play with it together, but trying to establish the edges should probably be a _non-nested_ for loop. In essence, make the vertices with one loop. Then in a totally separate for loop, establish the edges.